tests/network/setup_multimachine.pm

```diff
@@ -36,30 +36,33 @@ sub run {
     # Configure the internal network an  try it
     if ($hostname =~ /server|master/) {
         setup_static_mm_network('10.0.2.101/24');
-        #if server running opensuse.
+        #if server running openSUSE.
         if (is_opensuse) {
-            disable_and_stop_service('NetworkManager', ignore_failure => 1);
-            assert_script_run 'systemctl start  wicked';
+            assert_script_run 'systemctl restart  wicked';
+        }
+        #if server runing on SLED
```

Comment:
typo: s/runing/running/. And I suggest to follow the common style to have a space for separation after `#` and start uppercase. But actually in this case you might consider leaving out the comment, IMHO the code itself is readable enough :)

Reply:
Aff! Ok, changed. And double chek about # and Uppercas to commom style. Thanks